Update reading XMLTV xmltv_ns for PVR API v9.0.0

emveepee opened the pull request:
Update EpisodePartNumber with the total number of episodes in an episode sequence.  Use with Episode number

As an example from this XMLTV programme the programs shows E45/48 in the Estuary skin

```
	<programme start="20240823161500 +0000" stop="20240823170000 +0000" channel="nextpvr-7400">
		<title>Pointless</title>
		<desc>45/48. Alexander Armstrong is joined by co-host Desiree Burch for the quiz where contestants try to score as few points as possible by coming up with the answers no-one else could think of. [S] [HD]</desc>
		<category>Show / Game show</category>
		<episode-num system="xmltv_ns">..44/47</episode-num>
	</programme>
```
![image](https://github.com/user-attachments/assets/62a8830d-ef70-4e3b-96cb-40015aa2bf54)

phunkyfish reviewed (changes requested):
Thanks, comment looks good. 

One small request to split the code change from the changelog and version. For iptvsimple we add a separate comment for the version bump. 

Eg: https://github.com/kodi-pvr/pvr.iptvsimple/commit/76b7ea9dd6b1bb2b0c7ed41a3ade1f936da307d6

emveepee committed:
Update reading XMLTV xmltv_ns for PVR API v9.0.0

Update EpisodePartNumber with the total number of episodes in an episode sequence.  Use with Episode number
